src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -142,6 +147,106 @@ public class Xthor : BaseCachingWebIndexer
+
+            var ConfigWebRequestDelay = new SingleSelectConfigurationItem("Which delay do you want to apply between each requests made to tracker ?", new Dictionary<string, string>
+            {
+                {"2.1", "2.1s (minimum)"},
+                {"2.2", "2.2s"},
+                {"2.3", "2.3s"},
+                {"2.4", "2.4s" },
+                {"2.5", "2.5s"},
+                {"2.6", "2.6s"}
+            })
+            { Value = "2.1" };
+            ConfigData.AddDynamic("webRequestDelay", ConfigWebRequestDelay);
+
+            var ConfigMaxPages = new SingleSelectConfigurationItem("How many pages do you want to follow ?", new Dictionary<string, string>
+            {
+                {"1", "1 (32 results - default / best perf.)"},
+                {"2", "2 (64 results)"},
+                {"3", "3 (96 results)"},
+                {"4", "4 (128 results - hard limit max)" },
+            })
+            { Value = "1" };
+            ConfigData.AddDynamic("maxPages", ConfigMaxPages);
+
+            var ConfigMaxPagesBypassForTMDB = new BoolConfigurationItem("Do you want to bypass max pages for TMDB searches ? (Radarr) - Hard limit of 4") { Value = true };
+            ConfigData.AddDynamic("maxPagesBypassForTMDB", ConfigMaxPagesBypassForTMDB);
+
+            var ConfigEnhancedAnimeSearch = new BoolConfigurationItem("Do you want to use enhanced ANIME search ?") { Value = false };
+            ConfigData.AddDynamic("enhancedAnimeSearch", ConfigEnhancedAnimeSearch);
+
+            var ConfigMultiReplacement = new StringConfigurationItem("Do you want to replace \"MULTI\" keyword in release title by another word ?") { Value = "MULTI.FRENCH" };
+            ConfigData.AddDynamic("multiReplacement", ConfigMultiReplacement);
+
+            var ConfigSubReplacement = new BoolConfigurationItem("Do you want to replace \"VOSTFR\" and \"SUBFRENCH\" with \"ENGLISH\" word ?") { Value = false };
```

Comment:
change to match other indexers - `Do you want to replace "VOSTFR" and "SUBFRENCH" with "ENGLISH" word ?` > `Replace VOSTFR with ENGLISH`

Reply:
Code replace SUBFRENCH intentionally too as I seen

Reply:
Oh, true, missed that. How about `Replace VOSTFR and SUBFRENCH with ENGLISH`?